lib/vhost/vhost_user.c

```diff
@@ -1445,6 +1445,9 @@ vhost_user_set_mem_table(struct virtio_net **pdev,
 	if (dev->async_copy && rte_vfio_is_enabled("vfio"))
 		async_dma_map(dev, true);
 
+	if ((dev->vdpa_dev) && (dev->vdpa_dev->ops->set_mem_table))
+		dev->vdpa_dev->ops->set_mem_table(dev->vdpa_dev);
```

Comment:
good work to have explicit call, instead of doing implicit in the dev_config.

Reply:
I am not understand. Can you show me the code example?